addons/website_forum/static/src/js/website_forum.js

```diff
@@ -155,8 +155,15 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
                 // o_we_selected_image has not always been removed when
                 // saving a post so we need the line below to remove it if it is present.
                 $form.find('.note-editable').find('img.o_we_selected_image').removeClass('o_we_selected_image');
-                $form.on('click', 'button, .a-submit', () => {
+                $form.on('click', 'button, .a-submit', async (ev) => {
                     $form.find('.note-editable').find('img.o_we_selected_image').removeClass('o_we_selected_image');
+                    const $croppedImg = wysiwyg.$editor.find('.o_cropped_img_to_save');
+                    // this will call when only form is submiting
+                    if (ev.target.classList.contains('o_submit_btn') && $croppedImg.length) {
```

Comment:
Do you really need this double condition ? I think you can launch `saveCroppedImages` in any case
- If there is no '.o_cropped_img_to_save' - > `saveCroppedImages` won't do anything so I don't see the problem.
- I also dont understand why you needed to add the class `o_submit_btn`

Reply:
> Do you really need this double condition ? I think you can launch `saveCroppedImages` in any case
> 
> * If there is no '.o_cropped_img_to_save' - > `saveCroppedImages` won't do anything so I don't see the problem.
==> yes, we can but is it good to call every time with each click of the toolbar's button??

> * I also dont understand why you needed to add the class `o_submit_btn`

for about to add the class 'o_submit_btn', we avoiding to calling each time of the saveCroppedImages method, when we click on an editor toolbar button(style, bold, italic, underline, etc...). if cropped_image is existed or not

Reply:
ok :+1:

Reply:
Can't you just do it the opposite way?
Eg exclude the behavior for button which have `.note-toolbar` as parent, instead of adding a class on all submit button to differentiate it from toolbar button?

Reply:
@jpr-odoo